src/backend/buildSearchQuery.ts

```diff
@@ -25,12 +19,12 @@ export interface Field {
             field: SourceField
             size: number
         }
-        range?: {}
-        missing?: {}
-        date_histogram?: {}
+        range?: object
+        missing?: object
+        date_histogram?: object
     }
-    filterClause?: any
-    filterExclude?: any
+    filterClause?: object | null
```

Comment:
Its good practice to avoid generic object type, and use or write strongly typed interfaces. But if we want to just use this approach for the sake of finishing with the errors, its fine. On the long term we should convert to only strong typing our values in my opinion

Reply:
wanted to avoid that complexity, but maybe it will help sometime, added elasticsearch typings